test/tools/Modules/SelfSignedCertificate/SelfSignedCertificate.psd1

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Robert Holt.
```

Comment:
Why is this being added to the repo?  It's being pulled in by PSGalleryTestModules.csproj

Reply:
:-) It was your commit 2 week ago.